### DISCUSSION TITLE
Autocad R14 (circa 1995) on Windows 10 works, runs, but closes with any attempt at file read/write.

### DISCUSSION THREAD

**Discussion by Billsee** [otya128/winevdm#1396] [Q&A] [answered]
I am loading the Autocad CDROM with otvdmw.exe. It starts, then flashes a box with this message:

"C:\WINE\otvdm-v0.9.0\otvdm.exe
version: 2446
0AC4; fixme:module:LoadModule_wine-implementation
strange error set by CreateProcess: 740"

But it continues loading: Autocad always runs/draws (sometimes missing its toolbars...) but when I attempt to load or save a drawing file it says "heap error", then the Autocad fatal-error-message which asks if you want to save the current file before it closes.
I'm  NOT  a computer whiz: doing something really stupid in the installation process is a very real possibility. 

Billsee

**Comment by cracyc** [comment]
Autocad r14 isn't a win16 program so nothing can be done with winevdm to make it work. Maybe look at https://forums.autodesk.com/t5/previous-version-support/r14-on-windows-10/td-p/6231731 for a possible solution.

**Comment by Billsee** [comment, marked as answer]
I had already looked on that Autodesk site: it pointed to otvdm which I had already attempted as described in my original post. Finally threw in the towel and bought the Longbow Converter. It worked and an hour later my Autocard R14 was running again on this 64 bit Windows 10 machine.
Billsee
